boards/nxp/lpcxpresso55s06/pre_dt_board.cmake

```diff
@@ -4,8 +4,6 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-# Suppress "simple_bus_reg" on LPC boards as all GPIO ports use the same register.
-list(APPEND EXTRA_DTC_FLAGS "-Wno-simple_bus_reg")
 
 # Suppress "unique_unit_address_if_enabled" to handle the following overlaps:
 # - /soc/peripheral@40000000/syscon@0 & /soc/peripheral@40000000/gpio@0
```

Comment:
```suggestion
# - /soc/peripheral@40000000/syscon@0 & /soc/peripheral@40000000/gpio@4008c000/gpio@0
```

Do we still need this suppression? When I remove this locally I don't see a dtc warning